src/app/app.js

```diff
@@ -1,4 +1,6 @@
 // Special Debug Console Calls!
+var localStorage = window.localStorage || {};
```

Comment:
If it's a replacement that works, why not! My pov is mostly that having a "quick and easy" db-like (like localStorage) for little things, it's a plus (over nedb, for instance). So yeah.